add lines about zenodo community (#80)
galaxy-iuc/standards: docs/best_practices/tool_xml.rst

When tools contain output filters, tests should be included that verify
this filtering occurs. See `planemo docs <https://planemo.readthedocs.io/en/latest/writing_how_do_i.html#test-output-filters-work>`__ for more information.

Test data guidelines:

* Test data should be as small as possible while still being representative.
* Consider using subsets or downsampled versions of real datasets.
* If larger test files are unavoidable, they should be deposited in
  the `Galaxy Tool Developers <https://zenodo.org/communities/galaxy_tool_dev_sig>`__
  Zenodo community rather than uploaded directly to
  GitHub or stored in private Zenodo repositories.

> To upload test data to the Zenodo community:

Reviewer: It would be nice if you can also show how a Zenodo dataset can be used in the Galaxy tool XML test.

Author: like this? [99cc5eb](https://github.com/galaxy-iuc/standards/pull/80/commits/99cc5eb0186fa781fd57e0387b7869438f775edc)

Reviewer: Yes, thanks!

> To upload test data to the Zenodo community:

Reviewer 2: ```suggestion
To upload test data to the Zenodo community:

- Login to your account on Zenodo.
- Click on "New Upload".
- On top of the page, click on "Select a Community".
- Look for "Galaxy Tool Developers", using the search bar. Click on "Select."
- You have to accept "The 'Galaxy Tool Developers' curators will have access to **view and edit** your upload's metadata and files." for a successful submission.
- Upload your file(s).
- Get a DOI for your upload.
- Fill as much as information and metadata as possible for your dataset.
- A reviewer from this community, will review your request and reply soon. They maybe ask you to enrich your dataset submission with more metadata or details. This process may takes a few days.

If you already have a record on Zenodo and want to add it to the "Galaxy Tool Developer" community:

- Go to your record page on Zenodo.
- Scroll down until you find "" on the right side of the page.
- Click on the gear icon.
- Click on "Submit to Community".
- Look for "Galaxy Tool Developers", using the search bar. Click on "Select."
- You have to accept "The 'Galaxy Tool Developers' curators will have access to **view and edit** your upload's metadata and files." for a successful submission.
- A reviewer from this community, will review your request and reply soon. They maybe ask you to enrich your dataset submission with more metadata or details. This process may takes a few days.
```

When tools contain output filters, tests should be included that verify
this filtering occurs. See `planemo docs <https://planemo.readthedocs.io/en/latest/writing_how_do_i.html#test-output-filters-work>`__ for more information.

Test data guidelines:

* Test data should be as small as possible while still being representative.
* Consider using subsets or downsampled versions of real datasets.

> * If larger test files are unavoidable, they should be deposited in

Reviewer 3: ```suggestion
* If test files are over 1MB in size, they should be deposited in
```

  the `Galaxy Tool Developers <https://zenodo.org/communities/galaxy_tool_dev_sig>`__
  Zenodo community rather than uploaded directly to
  GitHub or stored in private Zenodo repositories.

To upload test data to the Zenodo community:

- Login to your account on Zenodo.
- Click on "New Upload".
- On top of the page, click on "Select a Community".
- Look for "Galaxy Tool Developers", using the search bar. Click on "Select."

> - You have to accept "The 'Galaxy Tool Developers' curators will have access to **view and edit** your upload's metadata and files." for a successful submission.

Reviewer 3: Here, some info about licensing. Zenodo by default selects CC4.0. If the uploader is using some other published/subsampled dataset, the licensing must be according to the original dataset.

Reviewer 3: - add appropriate keywords and subjects for easy findability

Author: [0c4d16d](https://github.com/galaxy-iuc/standards/pull/80/commits/0c4d16ddac535bb99423c24062f544db4d6e6f2e) ?

When tools contain output filters, tests should be included that verify
this filtering occurs. See `planemo docs <https://planemo.readthedocs.io/en/latest/writing_how_do_i.html#test-output-filters-work>`__ for more information.

Test data guidelines:

* Test data should be as small as possible while still being representative.
* Consider using subsets or downsampled versions of real datasets.
* If test files are over 1MB in size, they should be deposited in
  the `Galaxy Tool Developers <https://zenodo.org/communities/galaxy_tool_dev_sig>`__
  Zenodo community rather than uploaded directly to
  GitHub or stored in private Zenodo repositories.

To upload test data to the Zenodo community:

- Login to your account on Zenodo.
- Click on "New Upload".
- On top of the page, click on "Select a Community".
- Look for "Galaxy Tool Developers", using the search bar. Click on "Select."
- You have to accept "The 'Galaxy Tool Developers' curators will have access to **view and edit** your upload's metadata and files." for a successful submission.
- Upload your file(s).
- Get a DOI for your upload.

> - Fill as much as information and metadata as possible for your dataset.

Reviewer 4: ```suggestion
- Fill as much information and metadata as possible for your dataset.
```

> - A reviewer from this community, will review your request and reply soon. They maybe ask you to enrich your dataset submission with more metadata or details. This process may takes a few days.

Reviewer 4: ```suggestion
- A reviewer from this community will review your request and reply soon. They may ask you to enrich your dataset submission with more metadata or details.
```

> If you already have a record on Zenodo and want to add it to the "Galaxy Tool Developer" community:

Reviewer 5: I would prefer this procedure (i.e. user just uploads and then gives permissions). This way people can open PRs wo us needing to interact. Once the PR is ready they can give us access. 

Maybe we need CI to check if we have access to all linked items. Or an item in the PR template is sufficient.

Author: AFAIK, both will look the same to us. We only review the files inside and see if they are OK (by license, for example, or size or ...) Then we agree to accept the access.

Can we add CI to zenodo?

Reviewer 5: We can add a test to our CI iterating over all zenodo links in contained tools.

- Go to your record page on Zenodo.
- Scroll down until you find "Communities" on the right side of the page.
- Click on the gear icon.
- Click on "Submit to Community".
- Look for "Galaxy Tool Developers", using the search bar. Click on "Select."
- You have to accept "The 'Galaxy Tool Developers' curators will have access to **view and edit** your upload's metadata and files." for a successful submission.
- A reviewer from this community, will review your request and reply soon. They maybe ask you to enrich your dataset submission with more metadata or details. This process may takes a few days.

Please make note:

* Zenodo defaults license to CC BY 4.0; verify or change the licence at upload. If your

> file derives from another dataset, keep the original licence and note provenance.

Reviewer 4: ```suggestion
Please note:

* Zenodo defaults to the CC BY 4.0 license for new uploads; please verify or change the license if required. 

   In particular, if your file derives from another dataset, keep the original license and note provenance.
```

* Add concise keywords/subjects (community, format, domain, tool) for discoverability.

> 

Reviewer 4: I would prefer if we take out this entire section about zenodo community specifics and put it somewhere linkable on the zenodo community pages.
Then we can simply link out from here like we're doing for sections of the planemo docs, etc., and keep this document shorter.

Author: so I move them here?
https://zenodo.org/communities/galaxy_tool_dev_sig/about
